lib/widgets/profile.dart

```diff
@@ -73,17 +75,28 @@ class ProfilePage extends StatelessWidget {
           ),
           // TODO write a test where the user is muted; check this and avatar
           TextSpan(text: store.userDisplayName(userId, replaceIfMuted: false)),
+          UserStatusEmoji.asWidgetSpan(
+            userId: userId,
+            fontSize: 20,
```

Comment:
nit: this says `fontSize: 20` but the presence circle above says `fontSize: nameStyle.fontSize!`; should say the same thing both places, since the intent is for them to be the same